worker/runtime/container.go

```diff
@@ -103,6 +104,9 @@ func (c *Container) Run(
 
 	proc, err := task.Exec(ctx, id, &procSpec, cio.NewCreator(cioOpts...))
 	if err != nil {
+		if isNoSuchExecutable(err) {
+			return nil, garden.ExecutableNotFoundError{Message: err.Error()}
+		}
```

Comment:
I think this should be further down, on `proc.Start`. It looks like `Exec` only [sets up the process](https://godoc.org/github.com/containerd/containerd#Task) while `Start` actually [executes the binary](https://godoc.org/github.com/containerd/containerd#Process)

Reply:
Ah, thanks! Could've sworn I ran the new testflight tests against it, but must have forgotten to enable containerd or something. Fixed